### PR TITLE
Make dates human readable in the subsystem UIs

### PR DESCRIPTION
**Pull request by ckelleyRH**
Combination of #3980 and #3983 backported to `v10` branch.

### CODE REVIEW

File: base/tps/shared/webapps/tps/js/cert.js [hunk 1]
```diff
@@ -22,6 +22,7 @@
 var CertificateModel = Model.extend({
     urlRoot: "/tps/rest/certs",
     parseResponse: function(response) {
+    var respModifyTime = (response.ModifyTime == null) ? undefined : new Date(response.ModifyTime)
```
Comment (edewata) [inline base/tps/shared/webapps/tps/js/cert.js:25]: Probably should be indented.

**Review by edewata — APPROVED** [review]
A few minor comments, but overall it LGTM.

File: base/tps/shared/webapps/tps/js/cert.js (hunk 1 above)
Comment (ckelleyRH) [reply base/tps/shared/webapps/tps/js/cert.js:25]: Ack.

File: base/tps/shared/webapps/tps/js/cert.js [hunk 2]
```diff
@@ -62,6 +63,7 @@ var CertificateCollection = Collection.extend({
         return response.Link;
     },
     parseEntry: function(entry) {
+        custModifyTime = (entry.ModifyTime == null) ? undefined : new Date(entry.ModifyTime)
```
[the remark this answers is not in the document]
Comment (ckelleyRH) [reply base/tps/shared/webapps/tps/js/cert.js:66]: Ack.

**Comment by ckelleyRH** [comment]
@edewata - Err... I have pushed a fix but it has not updated the PR, it exists in the branch in my fork though... any idea why?

**Comment by edewata** [comment]
@ckelleyRH Hmm.. I have no idea. Maybe try rebasing the branch? If it still doesn't work you might need to contact GitHub support, or just push directly from CLI.

**Comment by ckelleyRH** [comment]
I just made a trivial change to the HEAD commit and force-pushed, seems to have done the trick. Thanks!

**Pull request merged**